samples/highcharts/accessibility/internationalization/demo.js

```diff
@@ -0,0 +1,140 @@
+const languages = {
+    no: {
+        lang: {
+            decimalPoint: ',',
+            thousandsSep: '.',
+            loading: 'Later­...',
+            months: ['Januar', 'Februar', 'Mars', 'April', 'Mai', 'Juni', 'Juli', 'August', 'September', 'Oktober', 'November', 'Desember'],
+            shortMonths: ['Jan', 'Feb', 'Mar', 'Apr', 'Mai', 'Jun', 'Jul', 'Aug', 'Sep', 'Okt', 'Nov', 'Des'],
+            weekdays: ['Mandag', 'Tirsdag', 'Onsdag', 'Torsdag', 'Fredag', 'Lørdag', 'Søndag­'],
+            exportButtonTitle: 'Eksporter',
+            printButtonTitle: 'Print',
+            printChart: 'Skriv ut diagram',
+            viewFullscreen: 'Se i fullskjerm',
+            rangeSelectorFrom: 'Fra',
+            rangeSelectorTo: 'Til',
+            rangeSelectorZoom: 'Zoom',
+            downloadPNG: 'Last ned som PNG-bilde',
+            downloadJPEG: 'Last ned som JPEG-bilde',
+            downloadPDF: 'Last ned som PDF-dokument',
+            downloadSVG: 'Last ned som SVG-bilde',
+            title: 'AAPL Aksjekurs'
```

Comment:
Instead of placing title under `no.lang`, you could just put it under `no.title.text`, and when calling `setOptions()` it will automatically go in the right place for the chart options.